.github/workflows/main.yml

```diff
@@ -79,13 +79,9 @@ jobs:
               PRIMARY: "substratevm"
           - env:
               JDK_VERSION: "21"
-              GATE_TAGS: "build,test,helloworld"
+              GATE_TAGS: "build,helloworld"
               PRIMARY: "substratevm"
               PIP_PACKAGES: "jsonschema==4.6.1"
-          - env:
-              JDK_VERSION: "21"
-              GATE_TAGS: "build,helloworld_debug"
```

Comment:
... and `debuginfotest` here?

Reply:
We already run `debuginfotest` in https://github.com/oracle/graal/blob/f658f09475f468ed9e50934267c44aa6e4c4987a/.github/workflows/main.yml#L85-L88